flink-ai-flow/ai_flow/scheduler_service/service/workflow_event_processor.py

```diff
@@ -41,30 +37,35 @@ def __init__(self, namespace, event_type, key, sender):
         self.sender = sender
 
 
-class WorkflowEventProcessor(Process):
-    def __init__(self, event_queue: Queue, store: AbstractStore, scheduler: Scheduler,
+class Poison(object):
+    pass
+
+
+class WorkflowEventProcessor:
+    def __init__(self, event_queue, store: AbstractStore, scheduler: Scheduler,
```

Comment:
The name of the first arg should be `conn` or sth else.

Reply:
fixed in ae568b0